packages/assets/package.json

```diff
@@ -11,7 +11,15 @@
   "homepage": "https://github.com/facebook/react-native/tree/HEAD/packages/assets#readme",
   "keywords": ["assets", "registry", "react-native", "support"],
   "bugs": "https://github.com/facebook/react-native/issues",
-  "engines": {
-    "node": ">=18"
-  }
+  "exports": {
+    "./package.json": "./package.json",
+    "./path-support": "./dist/path-support.js",
+    "./registry": "./dist/registry.js"
```

Comment:
To preserve backwards-compatibility, we'll need some additional `"exports"` subpaths to cover all cases.

> [!Note]
> The `build` script will rewrite all `./src/` exports to `./dist/`! Therefore please list paths to `./src/` only (committing before running `yarn build`).
>
> Within the React Native monorepo, this allows these built Node.js packages to run from source 🙂.

nit: Please also sort `./package.json` last.

```suggestion
    "./path-support": "./src/path-support.js",
    "./path-support.js": "./src/path-support.js",
    "./src/path-support": "./src/path-support.js",
    "./src/path-support.js": "./src/path-support.js",
    "./registry": "./src/registry.js",
    "./registry.js": "./src/registry.js",
    "./src/registry": "./src/registry.js",
    "./src/registry.js": "./src/registry.js",
    "./package.json": "./package.json"
```

But also see my other comment in `path-support.js` — if we know all package consumers, we could reduce this set of subpaths.

Reply:
Ah, of course! 👍🏻